packages/SwingSet/test/change-parameters/test-change-parameters.js

```diff
@@ -3,8 +3,8 @@ import { test } from '../../tools/prepare-test-env-ava.js';
 
 // eslint-disable-next-line import/order
 import { assert } from '@agoric/assert';
-import { getAllState } from '@agoric/swing-store';
-import { provideHostStorage } from '../../src/controller/hostStorage.js';
+// eslint-disable-next-line import/order
```

Comment:
Lint disablers for import order tend to accumulate because lint whines about import order unpredictably but never complains about being redundantly told to shut up.